dts/bindings/sensor/nxp,fxas21002-i2c.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2018, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: FXAS21002 3-axis gyroscope sensor
+
+compatible: "nxp,fxas21002"
```

Comment:
@galak @mbolivar-nordic, do we support defining 2 identical compatibles but with different properties? (see (*)) I see some other sensors doing that, so I guess they are merged altogether?

Reply:
Yes, this is fine. Note that including `i2c-device.yaml` below ensures that `on-bus: i2c` is in the final merged binding file. That combined with the `spi-device.yaml` include in the `nxp,fxas21002-spi.yaml` file creates two separate entries in the `devicetree.edtlib.EDT` object's `_compat2binding` map.

The purpose of this is precisely to allow for bus-specific properties for the same compatible.

This is documented in https://docs.zephyrproject.org/latest/build/dts/bindings.html#on-bus.